docs/CloudReady/20-fabric.md

```diff
@@ -37,31 +37,31 @@ just network-channel
 
 ```
 
-- Set the location for the network's TLS certificates, channel MSP, and user enrollments 
+- Set the location for the network's TLS certificates, channel MSP, and user enrollments
 ```shell
 
 export WORKSHOP_CRYPTO=$WORKSHOP_PATH/infrastructure/sample-network/temp
 
 ```
 
 
-## Post Checks 
+## Post Checks
 
 ```shell
 
 curl \
   -s \
   --cacert $WORKSHOP_CRYPTO/cas/org1-ca/tls-cert.pem \
-  https://${WORKSHOP_NAMESPACE}-org1-ca-ca.$WORKSHOP_INGRESS_DOMAIN/cainfo \
+  https://$WORKSHOP_NAMESPACE-org1-ca-ca.$WORKSHOP_INGRESS_DOMAIN/cainfo \
```

Comment:
I realize it is a good practice to use curly braces, but when I copy/pasted from the markdown to Mac terminal, it added backslash escapes in front of the curly braces, and the commands wouldn't work.